src/styles/blocks/block.scss

```diff
@@ -0,0 +1,31 @@
+.block {
+  height: 300px;
+  min-width: 300px;
```

Comment:
please follow BEM, do not set external sizes in blocks, create external block and write sizes inside elements

Reply:
I rewrite a code.